examples/alice_http_client/lib/main.dart

```diff
@@ -21,11 +22,8 @@ class _MyAppState extends State<MyApp> {
   late final AliceHttpClientAdapter _httpClientAdapter =
       AliceHttpClientAdapter();
 
-  late final Alice _alice = Alice(
-    showNotification: true,
-    showInspectorOnShake: true,
-    maxCallsCount: 1000,
-  )..addAdapter(_httpClientAdapter);
+  late final Alice _alice = Alice(configuration: AliceConfiguration())
```

Comment:
Since this is an example, declare the configuration in a separate variable, just so that it's clearer for a new user.

Reply:
👍